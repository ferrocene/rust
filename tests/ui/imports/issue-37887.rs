//@ revisions: edition2015 edition2021
//@[edition2015] edition: 2015
//@[edition2021] edition: 2021
//
fn main() {
    extern crate test; //~ ERROR use of unstable
    use test::*; //[edition2015]~ ERROR unresolved import
                 //[edition2021]~^ ERROR use of unstable
}
