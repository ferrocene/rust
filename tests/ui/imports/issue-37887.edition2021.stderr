error[E0658]: use of unstable library feature `test`
  --> $DIR/issue-37887.rs:6:5
   |
LL |     extern crate test;
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: see issue #50297 <https://github.com/rust-lang/rust/issues/50297> for more information
   = help: add `#![feature(test)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error[E0658]: use of unstable library feature `test`
  --> $DIR/issue-37887.rs:7:9
   |
LL |     use test::*;
   |         ^^^^
   |
   = note: see issue #50297 <https://github.com/rust-lang/rust/issues/50297> for more information
   = help: add `#![feature(test)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
